internal/composer-koji/openapi.go

```diff
@@ -0,0 +1,184 @@
+// Package Openapi provides primitives to interact the openapi HTTP API.
+//
+// Code generated by github.com/deepmap/oapi-codegen DO NOT EDIT.
+package Openapi
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"github.com/deepmap/oapi-codegen/pkg/runtime"
```

Comment:
It seems to me that this module isn't packaged in Fedora. Fun times ahead! Same for `go-chi` and `openapi3`.

Reply:
Ugh! Sorry about that, that was an oversight, I thought we had figured all that out. Thanks for catching that!

Reply:
I don't think we'll block on this. Let's land the code before worrying about packaging it. We could start out with only generating RHEL rpms, until we get the deps packaged for Fedora, as vendoring circumvents the issue.